README.md

```diff
@@ -25,7 +25,7 @@
     <span> | </span>
     <a href="https://docs.rs/crate/plotly/">API Docs</a>
     <span> | </span>
-    <a href="https://github.com/igiagkiozis/plotly/blob/master/CHANGELOG.md">Changelog</a>
+    <a href="https://github.com/igiagkiozis/plotly/blob/main/CHANGELOG.md">Changelog</a>
```

Comment:
Might also in this PR be worth changing the self-referential links to https://github.com/plotly/plotly.rs.

It looks like their are a bunch; same basic idea as changing the branch references.